feat: one more paragraph to explain `chatmail` (#127)
deltachat/chatmail: www/src/info.md

## More information 

> `nine.testrun.org` provides a low-maintenance, resource efficient and 
> interoperable e-mail service for everyone. What's behind a `chatmail` is 
> effectively a normal e-mail address just like any other but optimized 
> for the usage in chats, especially DeltaChat.

Reviewer: i find the "what's behind a 'chatmail'" part hard to understand.

is the gist the following?

```suggestion
`nine.testrun.org` provides a low-maintenance, resource efficient and 
interoperable e-mail service for everyone.
This is done by using "chatmail",
effectively a normal e-mail address just like any other but optimized 
for the usage in chats, especially DeltaChat.
```

Reviewer 2: i'll merge and we can further modify then. thanks.
